client/verta/verta/registry/errors.py

```diff
@@ -0,0 +1,18 @@
+class CustomModelError(Exception):
+    """Exception to be raised in user-defined models.
+
+    Messages of this class are caught in model containers and converted to
+    HTTP errors with the specified HTTP code and message in the response body.
+
+    Attributes
+    ----------
+    http_code : int
+        The http code to be returned by a wrapping service.
+    message : str
+        The response message.
+    """
+
+    def __init__(self, http_code, message):
```

Comment:
Would you know if it'd be more idiomatic to have the message first? I've rarely encountered exceptions that didn't have the `message` as the first parameter, but that might just be limited exposure on my part.